### PR TITLE
adds Logo to SeriesNavButton

### PR DESCRIPTION
**Pull request by ovbm**

### CODE REVIEW

File: components/Article/SeriesNavButton.js [hunk 1]
```diff
@@ -59,13 +62,9 @@ const styles = {
       fontSize: 18
     }
   }),
-  arrow: css({
```
Comment (tpreusse) [inline components/Article/SeriesNavButton.js:62]: `styles.arrow` ist still used, please remove if no longer needed or re-instate if accidentally deleted.

Comment (ovbm) [reply components/Article/SeriesNavButton.js:62]: done

File: components/Article/SeriesNavButton.js [hunk 2]
```diff
@@ -90,12 +89,31 @@ const SeriesNavButton = ({ t, series, router }) => {
         }}
       >
         <span {...styles.title}>
+          {series.logo && (
+            <>
+              <img
+                {...styles.logo}
+                src={series.logo}
```
Comment (tpreusse) [inline components/Article/SeriesNavButton.js:96]:
```js
import { imageResizeUrl } from 'mdast-react-render/lib/utils'
```

For the case that SVGs are not used: we should resize to 48 height (2x of actual height for retina):
```suggestion
                src={imageResizeUrl(series.logo, 'x48')}
```

The asset server will ignore resize instructions for SVG so we do not need to check here.

File: components/Article/SeriesNavButton.js [hunk 3]
```diff
@@ -90,12 +89,31 @@ const SeriesNavButton = ({ t, series, router }) => {
...
+                {...colorScheme.set(
+                  'display',
+                  series.logoDark ? 'displayLight' : 'block'
+                )}
+              />
+              {series.logoDark && (
+                <img
+                  {...styles.logo}
+                  src={series.logoDark}
```
Comment (tpreusse) [inline components/Article/SeriesNavButton.js:105]:
```suggestion
                  src={imageResizeUrl(series.logoDark, 'x48')}
```

**Review by tpreusse — APPROVED** [review]
General approval for series nav, one more thing: needs a resize to max 48 height in the case pngs or jpgs are used.

**Pull request merged**